post.hbs

```diff
@@ -64,32 +64,24 @@
             </section> }}
             </footer>
         </article>
-
-        <aside class="gh-readmore">
-            <div class="gh-readmore-inner w-full max-w-screen-xl mx-auto my-0">
-                <div class="gh-readmore-next">
-                    {{#next_post}}
-                        <a href="{{url}}">
-                            <i class="fa fa-chevron-left" aria-hidden="true"></i>
-                            &nbsp;
-                            <h4>
-                                {{title}}
-                            </h4>
-                        </a>
-                    {{/next_post}}
-                </div>
-                <div class="gh-readmore-prev">
-                    {{#prev_post}}
-                        <a href="{{url}}">
-                            <h4>
-                                {{title}}
-                            </h4>
-                            &nbsp;
-                            <i class="fa fa-chevron-right" aria-hidden="true"></i>
-                        </a>
-                    {{/prev_post}}
-                </div>
-            </div>
+        
+        {{!-- implementation of related posts --}}
+        <article>
```

Comment:
Can we stick to the use of the `aside` element? No need to include the `gh-readmore` class. (Do also delete all traces of that from the CSS files.)

Reply:
I used gh-readmore css for styling